code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -82,14 +82,15 @@
 	for(var/obj/item/reagent_containers/food/snacks/cheesewedge/cheese in range(1, src))
 		if(prob(10))
 			var/cap = CONFIG_GET(number/ratcap)
-			if(LAZYLEN(SSmobs.cheeserats) > cap)
+			if(LAZYLEN(SSmobs.cheeserats) >= cap)
 				visible_message("<span class='warning'>[src] carefully eats the cheese, hiding it from the [cap] mice on the station!</span>")
 				qdel(cheese)
 				return
-			SSmobs.cheeserats += src
-			new /mob/living/simple_animal/mouse/bred(loc)
+			var/mob/living/newmouse = new /mob/living/simple_animal/mouse(loc, TRUE, src)
```

Comment:
var/mob/living/simple_animal/mouse/newmouse = new(loc)
Why are you trying to pass along TRUE, src?

Reply:
Yeah that's my bad

Reply:
init args

Reply:
Thats not mobs init args :eyes: